site/src/components/SyntaxHighlighter/coderTheme.ts

```diff
@@ -0,0 +1,239 @@
+      token: "support.type",
+    },
+    {
+      foreground: "66d9ef",
+      fontStyle: " italic",
+      token: "support.class",
+    },
+    {
+      foreground: "f8f8f0",
+      background: "ff79c6",
+      token: "invalid",
+    },
+    {
+      foreground: "f8f8f0",
+      background: "bd93f9",
+      token: "invalid.deprecated",
+    },
+    {
+      foreground: "cfcfc2",
+      token: "meta.structure.dictionary.json string.quoted.double.json",
+    },
+    {
+      foreground: "6272a4",
+      token: "meta.diff",
+    },
+    {
+      foreground: "6272a4",
+      token: "meta.diff.header",
+    },
+    {
+      foreground: "ff79c6",
+      token: "markup.deleted",
+    },
+    {
+      foreground: "50fa7b",
+      token: "markup.inserted",
+    },
+    {
+      foreground: "e6db74",
+      token: "markup.changed",
+    },
+    {
+      foreground: "bd93f9",
+      token: "constant.numeric.line-number.find-in-files - match",
+    },
+    {
+      foreground: "e6db74",
+      token: "entity.name.filename",
+    },
+    {
+      foreground: "f83333",
+      token: "message.error",
+    },
+    {
+      foreground: "eeeeee",
+      token:
+        "punctuation.definition.string.begin.json - meta.structure.dictionary.value.json",
+    },
+    {
+      foreground: "eeeeee",
+      token:
+        "punctuation.definition.string.end.json - meta.structure.dictionary.value.json",
+    },
+    {
+      foreground: "8be9fd",
+      token: "meta.structure.dictionary.json string.quoted.double.json",
+    },
+    {
+      foreground: "f1fa8c",
+      token: "meta.structure.dictionary.value.json string.quoted.double.json",
+    },
+    {
+      foreground: "50fa7b",
+      token:
+        "meta meta meta meta meta meta meta.structure.dictionary.value string",
+    },
+    {
+      foreground: "ffb86c",
+      token: "meta meta meta meta meta meta.structure.dictionary.value string",
+    },
+    {
+      foreground: "ff79c6",
+      token: "meta meta meta meta meta.structure.dictionary.value string",
+    },
+    {
+      foreground: "bd93f9",
+      token: "meta meta meta meta.structure.dictionary.value string",
+    },
+    {
+      foreground: "50fa7b",
+      token: "meta meta meta.structure.dictionary.value string",
+    },
+    {
+      foreground: "ffb86c",
+      token: "meta meta.structure.dictionary.value string",
+    },
+  ],
+  colors: {
+    "editor.foreground": hslToHex(theme.palette.text.primary),
+    "editor.background": hslToHex(theme.palette.background.paper),
+    "editor.selectionBackground": hslToHex(theme.palette.action.hover),
+    "editor.lineHighlightBackground": hslToHex(
+      theme.palette.background.paperLight,
+    ),
+    "editorCursor.foreground": "#f8f8f0",
+    "editorWhitespace.foreground": "#3B3A32",
+    "editorIndentGuide.activeBackground": "#9D550FB0",
+    "editor.selectionHighlightBorder": "#222218",
+  },
+})
+
+export const useCoderTheme = (): { isLoading: boolean; name: string } => {
```

Comment:
Adding a comment here explaining what this hook/what its scope is might be helpful.

Reply:
Sure thing!